### DISCUSSION TITLE
Multi-Tenant building

### DISCUSSION THREAD

**Discussion by Starkery12** [microrealestate/microrealestate#191] [Ideas]
Hi,

For buildings with multiple appartments/tenants, you have to create an appartment for each one of the tenants, instead of just creating one building (with total rent), then just use it for multiple tenants (with their individual rent).

So I was wondering if it's possible to have "multi-tenant buildings", because current "building" type of propreties suppose the whole building will be rented to one person (which can be the case).

Another possible solution would be to add the option of "Copy from another building" (just like the one implemented for the tenant creation), that way creating appartments for same building will be easier to do, instead of repeating multiple times the same form filling.

Thank you !

**Comment by camelaissani** [comment]
Hi @Starkery12 

Currently, the app allows to create multiple appartments and rent them to different tenants. This should fulfill your need no?
What is missing is to attach the different appartments to one building.

Yes it might be more convenient to add a "Copy from another property" option to create multiple properties with similar information.

**Reply by Starkery12** [reply]
Yeah that fulfill the need, it's more of the "have to rewrite the adress and infos" for each one that is the issue.

If you could make a "Copy from another property" option that would be great !

Thanks for all you've done.

EDIT : I made a request issue for the "Copy from another property" : https://github.com/microrealestate/microrealestate/issues/192
